extensions/resteasy-classic/rest-client/runtime/pom.xml

```diff
@@ -56,6 +56,10 @@
                     <groupId>org.jboss.spec.javax.ws.rs</groupId>
                     <artifactId>jboss-jaxrs-api_3.0_spec</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>org.jboss.resteasy</groupId>
+                    <artifactId>resteasy-cdi</artifactId>
+                </exclusion>
```

Comment:
We are completely positive this is not needed? I wouldn't like breaking perfectly valid use cases to support a corner case.

Reply:
This should never have been here in the first place.

The CDI module provides access to beans that simply don't make sense when you only have the Client and not the Server